package.json

```diff
@@ -165,7 +169,8 @@
     "@babel/helpers": "^7.27.0",
     "cross-spawn": "^7.0.6",
     "micromatch": "^4.0.8",
-    "rollup": "^4.24.4"
+    "rollup": "^4.24.4",
+    "rxjs": "7.8.0"
```

Comment:
This does not appear to be used, does not appear to be a sub-dependency either 🤔